docs(Scheduler)documentation for ItemsPerSlot parameter (#2036)
telerik/blazor-docs: components/scheduler/views/timeline.md

In this article:

* [View Parameters](#view-parameters)
	* [Slots](#slots)
* [Example](#example)
* [Resource Grouping](#resource-grouping-in-the-timeline-view)

@[template](/_contentTemplates/scheduler/views.md#day-views-common-properties)
| `ColumnWidth` | `decimal` | The width of each time column in pixels.

> | `NumberOfDays` | `int` <br/> `1` | How many days will be shown side by side in the view.

Reviewer: ```suggestion
| `NumberOfDays` | `int` <br/> `1` | How many days to show side by side in the view.
```
